backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlApiMutationResult.java

```diff
@@ -21,12 +21,18 @@ public enum Status {
   private Status status;
   private Map<String, String> details = new LinkedHashMap<>();
   private String code;
+  private String taskId;
 
   public GraphqlApiMutationResult(Status status, String message, Object... formatValues) {
     this.status = status;
     this.message = String.format(message, formatValues);
   }
 
+  public GraphqlApiMutationResult setTaskId(String taskId) {
+    this.taskId = taskId;
+    return this;
+  }
+
```

Comment:
maybe use split into two separate mutation results ( with and without task id )

Reply:
solved on graphql level (which does have two different mutationresult)